src/darwin/Framework/CHIP/MTRDeviceControllerDelegate.h

```diff
@@ -54,6 +54,18 @@ API_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4))
  */
 - (void)controller:(MTRDeviceController *)controller commissioningComplete:(NSError * _Nullable)error;
 
+/**
+ * Notify the delegate when commissioning is completed.
+ * For getting commissioning controllee's deviceId
+ */
+- (void)controller:(MTRDeviceController *)controller
+    commissioningComplete:(NSError * _Nullable)error
+                 deviceId:(NSNumber * _Nullable)deviceId;
+
+/**
+ * Notify the delegate when read commissioning Infomation. (vendorID, productID)
+ */
+- (void)controller:(MTRDeviceController *)controller readCommissioningInfo:(NSDictionary<NSString *, id> * _Nullable)info;
```

Comment:
This needs to document what keys are expected to be present in which circumstances, what types the corresponding values should be, etc.

Given that this notification can't be acted on directly (because commissioning proceeds in parallel with the notification dispatch), what are the use cases for it?

Reply:
I will add the Key List.
As noted in the comments, the App can indicate which deviceType is connected to the User during Commissioning (through DCL). App requires Progress UI during Commissioning.

Reply:
@joonhaengHeo Please don't resolve comments until they are actually addressed....

The comments added:

1. Are mutually contradictory.
2. Don't make it clear what the key strings are.  The right thing to do there is probably to have extern strings for the keys, etc.  See the other places dictionaries are used in the API.
3. Don't define what the values are, like I requested above.

Most importantly: what is the use case here?  In practice, when using MatterSupport, commissioning via Matter.framework will always be on-network, so what is the use case for `wifiEndpointID` and `threadEndpointID`?

Reply:
1. When all conversations are resolved, I will change the implementation to use extern.
2. Added wifiEndpointID and threadEndpointID to be the same as Android Case.

Reply:
> Added wifiEndpointID and threadEndpointID to be the same as Android Case.

Being "the same as Android Case" is not a design criterion here.  Again, what is the use case for these things in practice?

Reply:
I think what the UI actually needs are VendorID and ProductID. Other values will be deleted after the conversation ends.

Reply:
I updated.